ethereum/networks/src/test/java/tech/pegasys/teku/networks/Eth2NetworkConfigurationTest.java

```diff
@@ -104,6 +107,36 @@ public void applyNamedNetworkDefaults_shouldOverwritePreviouslySetValues() {
     }
   }
 
+  @Test
+  public void bootnodesFromMainnetNetworkDefaults_CanBeParsed() {
+    final Eth2NetworkConfiguration config =
+        Eth2NetworkConfiguration.builder().applyMainnetNetworkDefaults().build();
+
+    List<NodeRecord> nodeRecords = parseBootnodes(config.getDiscoveryBootnodes());
+
+    assertTrue(nodeRecords.size() == config.getDiscoveryBootnodes().size());
```

Comment:
We try to use the `Assertions` library so this would be:
```
    assertThat(nodeRecords.size()).isEqualTo(config.getDiscoveryBootnodes().size());
```

And we could actually go a step further and check that the ENRs are still the same after being parsed and deserialised:

```suggestion
    assertThat(nodeRecords.stream().map(NodeRecord::asEnr))
        .containsExactlyInAnyOrderElementsOf(config.getDiscoveryBootnodes());
```

Reply:
I made the changes, and the test is more rigorous.